tests/topotests/bfd_vrf_ospf_unnumbered/r1/bfd-peers.json

```diff
@@ -0,0 +1,54 @@
+[
+  {
+    "multihop":false,
+    "peer":"2.2.2.2",
```

Comment:
Please use https://www.rfc-editor.org/rfc/rfc5737.html ranges or private ranges for topotests.